src/rdkafka_admin.c

```diff
@@ -2832,37 +2834,42 @@ rd_kafka_ConfigResource_add_ConfigEntry(rd_kafka_ConfigResource_t *config,
         rd_list_add(&config->config, entry);
 }
 
-
 rd_kafka_resp_err_t
-rd_kafka_ConfigResource_add_config(rd_kafka_ConfigResource_t *config,
+rd_kafka_ConfigResource_set_config(rd_kafka_ConfigResource_t *config,
                                    const char *name,
                                    const char *value) {
         if (!name || !*name || !value)
                 return RD_KAFKA_RESP_ERR__INVALID_ARG;
 
-        return rd_kafka_admin_add_config0(&config->config, name, value,
-                                          RD_KAFKA_ALTER_OP_ADD);
+        return rd_kafka_admin_add_config0(&config->config, name, value);
 }
 
-rd_kafka_resp_err_t
-rd_kafka_ConfigResource_set_config(rd_kafka_ConfigResource_t *config,
-                                   const char *name,
-                                   const char *value) {
-        if (!name || !*name || !value)
-                return RD_KAFKA_RESP_ERR__INVALID_ARG;
 
-        return rd_kafka_admin_add_config0(&config->config, name, value,
-                                          RD_KAFKA_ALTER_OP_SET);
-}
+rd_kafka_error_t *rd_kafka_ConfigResource_set_incremental_config(
```

Comment:
This should be `rd_kafka_ConfigResource_add_incremental_config`

Reply:
Currently it'using the same naming as rd_kafka_ConfigResource_set_config, the key currently is (name), in future could be (name, op_type)

Reply:
But it's adding incremental alter config to the config list of the resource. Set feels that it replaces the old one and set the current one.